dev/drivers/scripts/plots/cam/jevs_cam_snowfall_plots.sh

```diff
@@ -48,7 +50,7 @@ evs_ver_2d=$(echo $evs_ver | cut -d'.' -f1-2)
 # Developer Settings
 export envir=prod
 export DATAROOT=/lfs/h2/emc/stmp/$USER/evs_test/$envir/tmp
-export COMIN=/lfs/h2/emc/vpppg/noscrub/$USER/$NET/$evs_ver_2d
+export COMIN=/lfs/h2/emc/vpppg/noscrub/emc.vpppg/$NET/$evs_ver_2d
```

Comment:
lines 38 and 53 need to be reverted to previous definitions

Reply:
Fixed.

Reply:
Now it's done.